dolphinscheduler-ui/src/views/resource/components/resource/upload/use-form.ts

```diff
@@ -18,13 +18,17 @@
 import { reactive, ref, unref } from 'vue'
 import { useI18n } from 'vue-i18n'
 import type { FormRules } from 'naive-ui'
+import { ResourceType } from "@/views/resource/components/resource/types";
 
 const defaultValue = () => ({
+  id: -1,
   name: '',
   file: '',
   description: '',
+  type: undefined as unknown as ResourceType,
   pid: -1,
-  currentDir: '/'
+  currentDir: '/',
+  isReupload: false
 })
```

Comment:
It is better to define an interface for the defaultValue.

Reply:
> It is better to define an interface for the defaultValue.

resolved